src/components/Song/SongPresentation.js

```diff
@@ -99,7 +103,10 @@ function SongPresentation({ entry }) {
             </div>
             {hasMedia && (
               <div className="flex md:hidden mt-2">
-                {getMedia({ pictures: entry?.pictures, videos: entry?.videos })}
+                {getMedia({
+                  pictures: entry?.coverVisual,
```

Comment:
Have you considered what will happen to songs with multiple images? Where will those images be displayed?

This could be addressed when we convert to the new BE if you prefer. 
Otherwise this all good 👍

Reply:
I'll merge this now and we can address it with the new BE